src/hotspot/share/gc/shenandoah/shenandoahRootVerifier.cpp

```diff
@@ -30,6 +30,7 @@
 #include "code/codeCache.hpp"
 #include "gc/shenandoah/shenandoahAsserts.hpp"
 #include "gc/shenandoah/shenandoahHeap.hpp"
+#include "gc/shenandoah/shenandoahHeap.inline.hpp"
```

Comment:
I think you can drop the `#include "gc/shenandoah/shenandoahHeap.hpp"` then, as `.inline.hpp` is supposed to transitively include it.

Reply:
Good idea. 
Updated.
Thanks.